OFEP-ofo-flagd-service.md

```diff
@@ -0,0 +1,14 @@
+## flagd service deployment driven by OpenFeature Operator
+
+## State: DRAFTING
+
+Currently, OpenFeature Operator (OFO) manages the deployment of flagd by appending it to a pod's containers (sidecar pattern), thereby allowing containers within the pod to route to it. This OFEP campaigns for the extension of OFO to manage the deployment of externally accessible flagd (e.g. from client side applications, or even server side applications that don't want to use the sidecar pattern).
+
+## Background
+
+The driving force behind this is to simplify the deployment of flagd for use by client side applications. An [OFEP was drafted](./OFEP-ofo-flagd-client-support.md) to achieve this but subsequently withdrawn (reasons noted within). This OFEP recognises the limitations of the previous, presenting a modular solution with a broader scope.
+
+## Proposal
+
+Introduce a FlagdService custom resource definition (CRD) and controller.
+The controller uses the configuration defined within the custom resource (CR) to create a Service and a Deployment of flagd (backed by the Service). This is a common deployment pattern permitting access by any component that routes to the created Service (e.g. Ingress/Load Balancer).
```

Comment:
You might want to very briefly contrast this with the "injected sidecar" model (flagd+fronting service vs flagd injected into workload pod).

Reply:
I like the suggestion of @toddbaert
I would suggest highlighting as part of this OFEP the new set of RBAC required to introduce this.  
@skyerus do you plan to allow the CR to specify also the ns in which the deployment & service are created?